yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -127,6 +132,10 @@ func init() {
 	assessMigrationCmd.Flags().StringVar(&assessmentMetadataDirFlag, "assessment-metadata-dir", "",
 		"Directory path where assessment metadata like source DB metadata and statistics are stored. Optional flag, if not provided, "+
 			"it will be assumed to be present at default path inside the export directory.")
+
+	assessMigrationCmd.Flags().Int64Var(&sleepIntervalForIOPS, "sleep-interval", 120,
```

Comment:
I think `sleep-interval` is unclear for an arg name. It's not obvious why assess-migration would need to sleep.  We might need to think of a more specific argument name. Let's discuss this in standup!

Reply:
Yes, I also couldn't think of better alternative.

Reply:
Lets rename it to `--iops-measurement-interval` until we have something better.